django_project_base/account/rest/project_profiles.py

```diff
@@ -141,12 +140,13 @@ def create(self, request: Request, *args, **kwargs) -> Response:
         if "password" in request.data:
             request.data["password"] = make_password(request.data["password"])
         response = super().create(request, *args, **kwargs)
+        user = Profile.objects.get(pk=response.data["id"])
```

Comment:
user pride že kot parameter not?

Reply:
current user, da, tukaj pa fetchamo userja ki ga kreiramo.